src/api/java/baritone/api/utils/ExampleBaritoneControl.java

```diff
@@ -510,7 +511,8 @@ public boolean isInGoal(int x, int y, int z) {
                 baritone.getMineProcess().mine(quantity, block);
                 logDirect("Will mine " + quantity + " " + blockTypes[0]);
                 return true;
-            } catch (NumberFormatException | ArrayIndexOutOfBoundsException | NullPointerException ex) {}
+            } catch (NumberFormatException | ArrayIndexOutOfBoundsException | NullPointerException ex) {
```

Comment:
🤢

Reply:
I think that was changed when i used intellij reformat, it changed this a lot in files i didn't commit